geode-core/src/main/java/org/apache/geode/management/internal/cli/functions/DestroyDiskStoreFunction.java

```diff
@@ -80,7 +80,6 @@ public void execute(FunctionContext context) {
     } catch (VirtualMachineError e) {
```

Comment:
I know this is not your change, but would be nice to send an error message back all the time, instead of having the null message, the above catch block could be abbreviated to:
catch (IllegalStateException  | CacheClosedException ex) {
       CliFunctionResult result = new CliFunctionResult(memberId, false, ex.getMessage());		       
       context.getResultSender().lastResult(result);
} catch (VirtualMachineError e) 
.....

Reply:
I'm going to do that in the next PR which adds the `--if-exists` option.